fusor-ember-cli/app/routes/review/summary.js

```diff
@@ -4,9 +4,7 @@ export default Ember.Route.extend({
 
   model() {
     var deployment_id = this.modelFor('deployment').get('id');
-    return this.store.query('deployment', {search: "id = " + deployment_id}).then(function(results) {
```

Comment:
this is another workaround that we shouldn't need. It was needed to show the "Next" button after the deployment reaches 100%